### PR TITLE
Core: Fix ability to use component-level play functions

### PR DESCRIPTION
**Pull request by wKich**
Issue: Nope

## What I did
Fix ability to use `play` function defined in Meta level

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: lib/store/src/csf/prepareStory.ts [hunk 1]
```diff
@@ -194,7 +194,8 @@ export function prepareStory<TFramework extends AnyFramework>(
 
     return decoratedStoryFn(finalContext);
   };
-  const playFunction = storyAnnotations.play;
+  const playFunction =
+    storyAnnotations.play || componentAnnotations.play || projectAnnotations.play;
```
Comment (yannbf) [inline lib/store/src/csf/prepareStory.ts:198]: The play function is not included in the project level, can you remove it?
```suggestion
    storyAnnotations.play || componentAnnotations.play;
```

**Comment by shilman** [comment]
@wKich getting a typescript error in CI. can you update the types?

```
ERR! FAILED (ts) : src/csf/prepareStory.ts:198:51 - error TS2339: Property 'play' does not exist on type 'NormalizedComponentAnnotations<TFramework>'.

198     storyAnnotations.play || componentAnnotations.play;
```

**Comment by wKich** [comment]
Hm... It looks like `@storybook/csf` was changed

**Comment by wKich** [comment]
@shilman Should I revert that commit https://github.com/ComponentDriven/csf/commit/85efd3c2c619bd84cc876123b56662982cc6c3ed ?

**Comment by ndelangen** [comment]
@shilman @yannbf could you put this on your todo-list to review and or respond? 👏👏👏

**Review by shilman — APPROVED** [review]
LGTM. I think it was @tmeasday who might've had objections to the component-level play function

**Comment by tmeasday** [comment]
I don't have an issue unless I am forgetting something.

File: scripts/package.json [hunk 2]
```diff
@@ -3,16 +3,16 @@
   "version": "7.0.0-alpha.16",
   "private": true,
   "scripts": {
+    "docs:prettier:check": "cd ../docs && prettier --check ./snippets",
```
Comment (yannbf) [inline scripts/package.json:6]: Not sure why this change happened in the PR but I guess it makes sense, with the scripts being ordered alphabetically

**Comment by yannbf** [comment]
Merging this as the failure is unrelated and has been fixed in `next`!
